packages/studio/src/components/AssetSelectorItem.tsx

```diff
@@ -141,13 +160,53 @@ export const AssetFolderTree: React.FC<{
 	level: number;
 	tabIndex: number;
 	toggleFolder: (folderName: string, parentName: string | null) => void;
-}> = ({item, level, name, parentFolder, toggleFolder, tabIndex}) => {
+	dropLocation: string | null;
+	setDropLocation: React.Dispatch<React.SetStateAction<string | null>>;
+}> = ({
+	item,
+	level,
+	name,
+	parentFolder,
+	toggleFolder,
+	tabIndex,
+	dropLocation,
+	setDropLocation,
+}) => {
 	const combinedParents = useMemo(() => {
 		return [parentFolder, name].filter(NoReactInternals.truthy).join('/');
 	}, [name, parentFolder]);
-
+	const dragDepthRef = useRef(0);
+	const isDropDiv = useMemo(() => {
+		return dropLocation === combinedParents;
+	}, [combinedParents, dropLocation]);
+	useEffect(() => {
+		if (dropLocation === null) {
+			dragDepthRef.current = 0;
+		}
+	}, [dropLocation]);
 	return (
-		<div>
```

Comment:
https://github.com/remotion-dev/remotion/assets/1629785/28c1c086-43e0-4792-a9fd-367465fb9b40


It's weird that you can drop the file into a folder, but not over a folder. I thought it was a bug that all folders get highlighted at first, it's even weirder when the folders are expanded.